io.github.dida_code.OpstaKultura.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.dida_code.OpstaKultura",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "opstakultura",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "opstakultura",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/dida-code/opstakultura",
+                    "tag" : "v1.1.1"
```

Comment:
please add the commit for the tag. This protect against retagging. (which you did)